Source/Core/DolphinQt/Debugger/CodeDiffDialog.cpp

```diff
@@ -89,12 +94,19 @@ void CodeDiffDialog::CreateWidgets()
   m_matching_results_table->setColumnWidth(2, 4);
   m_matching_results_table->setColumnWidth(3, 210);
   m_matching_results_table->setColumnWidth(4, 65);
+  m_save_btn = new QPushButton(tr("Save"));
+  m_save_btn->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
+  m_load_btn = new QPushButton(tr("Load"));
+  m_load_btn->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
   m_reset_btn = new QPushButton(tr("Reset All"));
   m_reset_btn->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
   m_help_btn = new QPushButton(tr("Help"));
   m_help_btn->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
   auto* help_reset_layout = new QHBoxLayout;
   help_reset_layout->addWidget(m_reset_btn, 0, Qt::AlignLeft);
+  help_reset_layout->addStretch();
+  help_reset_layout->addWidget(m_save_btn, 0, Qt::AlignRight);
+  help_reset_layout->addWidget(m_load_btn, 0, Qt::AlignRight);

```

Comment:
Minor nitpick, but perhaps we rename `help_reset_layout` to `bottom_controls_layout`